opentdf-dev.yaml

```diff
@@ -77,6 +77,7 @@ server:
       #
       #  [matchers]
       #  m = g(r.sub, p.sub) && globOrRegexMatch(r.res, p.res) && globOrRegexMatch(r.act, p.act) && globOrRegexMatch(r.obj, p.obj)
+      userinfoEnrichment: false # Enables OIDC UserInfo enrichment (default: false)
```

Comment:
Does this setting go along with the user info cache to?

Reply:
I actually need to remove this. We always should call I think for casbin.